### PR TITLE
[Distribution][Plugin] Audio recorder

### PR DESCRIPTION
**Pull request by pitrackster**
Quite the same as video-recorder but for audio only.

**After GetUserMedia acceptance**
gum_ok

**Recording**
recording_started

**Recording ended (after max recording time was reached)**
recording_ended_no_record_selected

**One record has been selected, OK button is enabled**
recording_ended

**The File Resource created**

created_resource

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
     "name": "claroline/distribution",
-    "replace": {"claroline/distribution-bundle": "self.version"},
+    "replace": {
```
Comment (stefk) [inline composer.json]: Actually that property is overwritten by another `replace` below, so we could move `"claroline/distribution-bundle": "self.version"` there and delete this one.

This bundle and the video recorder bundle should also be added to the `replace` section.

File: plugin/audio-recorder/Controller/AudioRecorderController.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    /**
+     * @Route("/update/configuration/{id}", name="audio_recorder_config_save")
+     * @ParamConverter("config", class="InnovaAudioRecorderBundle:AudioRecorderConfiguration")
+     * @Method("POST")
+     */
+    public function updateConfigurationAction(AudioRecorderConfiguration $config, Request $request)
+    {
+        if ($request->isMethod('POST')) {
```
Comment (stefk) [inline plugin/audio-recorder/Controller/AudioRecorderController.php]: That check is not necessary (the @method annotation prevents the execution of the controller for methods other than POST).

File: plugin/audio-recorder/Entity/AudioRecorderConfiguration.php [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+   * @ORM\Column(name="max_recording_time", type="integer", options={"default" = 0})
+   */
+  protected $maxRecordingTime;
+
+  /**
+   * @var int
+   * Define maximum number of try
+   * @ORM\Column(name="max_try", type="integer", options={"default" = 0})
+   */
+  protected $maxTry;
```
Comment (stefk) [inline plugin/audio-recorder/Entity/AudioRecorderConfiguration.php]: Indentation issues in these 3 properties.

File: plugin/audio-recorder/VERSION.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+dev-master
```
Comment (stefk) [inline plugin/audio-recorder/VERSION.txt]: I think this file can be removed.

**Comment by stefk** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Comment by stefk** [comment]
@pitrackster, do we let the tests out of the suite and merge the PR as is?

**Comment by pitrackster** [comment]
@stefk yep it's ok for me !

**Comment by pitrackster** [comment]
shall we delete video-recorder and audio-recorder branches ?

**Comment by stefk** [comment]
@pitrackster, done.

**Comment by pitrackster** [comment]
@stefk great ! shall I delete audio-recorder branch ?

**Comment by stefk** [comment]
I already did it ;)

**Comment by pitrackster** [comment]
@stefk thanks ! :sleeping:

**Pull request merged**